README.markdown

```diff
@@ -136,35 +114,51 @@ This will ensure the version `1.4.0` of the puppet-agent package is installed. F
 
 ##### `arch`
 
-The architecture version you wish to install. Defaults to `$::architecture`. This parameter is [ignored](#known-issues) in Windows Server 2003.
+The architecture version you wish to install. Defaults to `$::facts['architecture']`.
+``` puppet
+  arch => 'x86_64'
+```
 
 ##### `collection`
 
-The Puppet Collection to track. Defaults to `PC1`.
+The Puppet Collection to track, should be one of `puppet5` or `puppet6`. Puppet collections contain the latest agents included
+in the collection's series, so the latest 5 series in puppet5 (for example: 5.5.10) and the latest 6 series in puppet6 (for
+example: 6.1.0).
+``` puppet
+  collection => 'puppet6'
+```
 
 ##### `is_pe`
 
 Install from Puppet Enterprise repos. Enabled if communicating with a PE master.
+``` puppet
+  is_pe => true
+```
 
 ##### `manage_repo`
 
 Boolean to determine whether to configure zypper/yum/apt/solaris repositories. Defaults to `true`.
 If set to false, it is assumed an internally hosted repository will be used for the installation,
 and the native package providers will be used to query pre-configured repos on the host being upgraded.
-
-##### `package_name`
-
-The package to upgrade to, i.e., `puppet-agent`. Currently, the default and only accepted value is `puppet-agent`.
+``` puppet
+  manage_repo => true
+```
 
 ##### `package_version`
 
 The package version to upgrade to. When upgrading from Puppet < 4.0, defaults to the puppet master's latest supported version
```

Comment:
Upgrading from < 4.0 is no longer supported in master -- we can probably just remove that part of this sentence